### DISCUSSION TITLE
What is the best programming language to study as your first language?

### DISCUSSION THREAD

**Discussion by AnzhiZhang** [Schweinepriester/github-profile-achievements#675] [Q&A] [answered]
C? Python? Java?
or even Scratch?

**Comment by guppy57** [comment, marked as answer]
I would study Python to understand how programming works on a high level. Yet, after a decent understanding, I would recommend you learn Object Orientated Programming and static types. The way I would learn these would be through studying Java. (even though I never use Java now, an introduction to it set me up for a much deeper understanding of programming)

After all of this, if you are interested in web development (like myself), I would transition into learning HTML, CSS, and JavaScript (in that order). These three languages make up the web!
